.github/dependabot.yml

```diff
@@ -11,3 +11,6 @@ updates:
       - "jpalvarezl"
       - "rmeissner"
     open-pull-requests-limit: 3
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-patch"]
```

Comment:
My take on this: this is better to go in a separate PR. It's a rule that applies to all dependencies but the PR/Commit is `Bump serde from 1.0.127 to 1.0.128`